routes/api/contacts.js

```diff
@@ -1,32 +1,35 @@
 const express = require("express");
 const router = express.Router();
-const contactsController = require("../../controllers/contactsCtrl");
-const validation = require("../../middlewares/contactsValidation");
+const {
+  getAll,
+  getOne,
+  addOne,
+  deleteOne,
+  changeData,
+  changeValueOfFavorite,
+} = require("../../controllers/contactsCtrl");
+
+const {
+  validateCreationOrUpdate,
+  validateStatusUpdate,
+} = require("../../middlewares/contactsValidation");
+
 const authMiddleware = require("../../middlewares/auth");
 
-router.get("/", authMiddleware, contactsController.getAll);
+router.get("/", authMiddleware, getAll);
```

Comment:
mysle, ze `authMiddleware` tez powinien byc lepiej nazwany. `authenticateUser`? (wiemy, ze to middleware, bo wpada jako druga funkcja w route (kazdy koder uzywajacy Express musi tak robic, wiec mozemy smialo zalozyc, ze ta czesc jest oczywista i nie wymaga wyjasnienia)).